content/en/news/releases/1.6.x/announcing-1.6.6/index.md

```diff
@@ -9,6 +9,10 @@ aliases:
     - /news/announcing-1.6.6
 ---
 
+{{< warning >}}
+This release contains a regression from 1.6.5 that prevents endpoints that are not associated with pods from working. Please upgrade to 1.6.7 instead.
```

Comment:
```suggestion
This release contains a regression from 1.6.5 that prevents endpoints not associated with pods from working. Please upgrade to 1.6.7 instead.
```

Reply:
Also, since 1.6.7 isn't yet available, to we want to qualify the last sentence? Maybe `Please upgrade to 1.6.7 when available`. Not sure if that should be changed post 1.6.7 or not.